app/css/components/mentor/representer-row.css

```
@@ -0,0 +1,92 @@
+.c-mentor-inbox,
+.c-mentor-queue {
+    & .--representer {
+        @apply border-backgroundColorC border-t-1;
+        @apply flex items-center;
+        @apply py-8 px-20;
+
+        &:hover {
+            @apply bg-bgGray;
+        }
+        &:first-child {
+            @apply border-t-0;
+        }
+
+        & .c-track-icon {
+            height: 32px;
+            width: 32px;
+            @apply mr-24;
+        }
+        & .c-exercise-icon {
+            height: 48px;
+            width: 48px;
+
+            @apply mr-24;
+        }
+
+        & .c-avatar {
+            height: 48px;
+            width: 48px;
+
+            @apply mr-16;
+        }
+        & .--info {
+            @apply flex-grow;
+
+            & .--exercise-title {
+                @apply flex items-center;
+                @apply font-medium text-16 text-textColor2 leading-160;
+                & .dot,
+                & .c-icon {
+                    @apply ml-8;
+                }
+
+                & .favorited {
+                    height: 22px;
+                    width: 22px;
+                }
+                & .previously-mentored {
+                    height: 18px;
+                    width: 18px;
+                    filter: var(--orange-filter);
+                }
+            }
+            & .--track-title {
+                @apply text-14 text-textColor6 leading-150;
+            }
+        }
+
+        & .--status {
+            @apply font-medium text-14 text-textColor6 leading-code;
+            @apply py-4 px-12 border-borderLight border-1 rounded-100;
+            @apply ml-56;
+        }
+        & time,
+        .--occurencies {
+            @apply font-medium text-14 text-textColor6 leading-code;
+            @apply text-right;
+            width: 140px;
+        }
+        & time {
+            @apply ml-32;
+        }
+
+        & .--feedback-glimpse {
+            @apply text-right truncate mr-60 leading-150 text-textColor6;
+            width: 25rem;
+        }
+
+        & .action-icon {
+            @apply ml-36 text-textColor6;
+
+            height: 16px;
+            width: 16px;
+        }
+    }
+}
+.--most-popular {
```

Generally nothing should be at global scope unless it's prefixed with `c-` (for component).

I think what you actually want here is:
```suggestion
  &.--most-popular {
```

---

Actually, from reading code further, this should just be nested inside the row in the CSS, it's not a modifier.

---

I moved this outside, because I reused this tag in somewhere completely different place, but I'll find a better way of solving this.